subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/fixture/AbstractTestFilteringIntegrationTest.groovy

```diff
@@ -368,6 +374,10 @@ abstract class AbstractTestFilteringIntegrationTest extends MultiVersionIntegrat
         output.contains('CTest!')
     }
 
+    protected ignoreWhenDryRun() {
```

Comment:
Tests that are making assertions against outputs of tests are doesn't make sence since there is no test execution during dryRun